File: core/src/si/feri/slidegame/assets/AssetDescriptors.java
package si.feri.slidegame.assets;

import com.badlogic.gdx.assets.AssetDescriptor;
import com.badlogic.gdx.scenes.scene2d.ui.Skin;

public class AssetDescriptors {
    public static final AssetDescriptor<Skin> UI_SKIN =
            new AssetDescriptor<>(AssetPaths.UI_SKIN, Skin.class);

    //
    //
    //

    private AssetDescriptors() {
    }
}

File: core/src/si/feri/rrizemljevid/assets/AssetDescriptors.kt
package si.feri.rrizemljevid.assets

import com.badlogic.gdx.assets.AssetDescriptor
import com.badlogic.gdx.graphics.g2d.TextureAtlas
import com.badlogic.gdx.scenes.scene2d.ui.Skin

object AssetDescriptors {
    val UI_SKIN = AssetDescriptor(AssetPaths.UI_SKIN, Skin::class.java)
    val GAMEPLAY = AssetDescriptor(AssetPaths.GAMEPLAY, TextureAtlas::class.java)
}